required/latex-lab/latex-lab-mathpkg.dtx

```diff
@@ -0,0 +1,107 @@
+% \iffalse meta-comment
+%
+%% File: latex-lab-mathpkg.dtx
+%
+% Copyright (C) 2022,2023 The LaTeX Project
+%
+% It may be distributed and/or modified under the conditions of the
+% LaTeX Project Public License (LPPL), either version 1.3c of this
+% license or (at your option) any later version.  The latest version
+% of this license is in the file
+%
+%    https://www.latex-project.org/lppl.txt
+%
+%
+% The development version of the bundle can be found below
+%
+%    https://github.com/latex3/latex2e/required/latex-lab
+%
+% for those people who are interested or want to report an issue.
+%
+%<*driver>
+\documentclass{l3doc}
+\EnableCrossrefs
+\CodelineIndex
+\begin{document}
+  \DocInput{latex-lab-mathpkg.dtx}
+\end{document}
+%</driver>
+%
+% \fi
+%
+%
+% \title{The \texttt{latex-lab-mathpkg} code\thanks{}}
+% \author{\LaTeX{} Project}
+%
+% \maketitle
+%
+% \newcommand\fmi[1]{\begin{quote} TODO: \itshape #1\end{quote}}
+% \newcommand\NEW[1]{\marginpar{\mbox{}\hfill\fbox{New: #1}}}
+% \providecommand\class[1]{\texttt{#1.cls}}
+% \providecommand\pkg[1]{\texttt{#1}}
+% \providecommand\hook[1]{\texttt{#1}}
+%
+% \begin{abstract}
+% \end{abstract}
+%
+% \tableofcontents
+%
+% \section{Introduction}
+%
+% This file implements adaptions to various math packages needed for 
+% the tagging project. 
+%
+% \MaybeStop{\setlength\IndexMin{200pt}  \PrintIndex  }
+%
+% \section{The Implementation}
+%
+%    \begin{macrocode}
+%<@@=math>
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+%<*kernel>
+%    \end{macrocode}
+%
+% \subsection{File declaration}
+%    \begin{macrocode}
+\ProvidesFile{latex-lab-mathpkg.ltx}
+        [2023-01-05 v0.1a mathpkg adaptions]
+%    \end{macrocode}
+% \subsection{Tagpdf support}
+% To make the code independent from tagging being loaded and active
+% we load the \pkg{tagpdf-base} package:
+%
+%    \begin{macrocode}
+\RequirePackage{tagpdf-base}
+%    \end{macrocode}
+% 
+%
+%    \begin{macrocode}
+\AddToHook{package/breqn/after}{
+   \RegisterMathEnvironment{dmath}
+   \RegisterMathEnvironment{dgroup*}
+   \AddToHook{env/dmath/begin}{\tagpdfparaOff}
+   \AddToHook{env/dgroup*/begin}{\tagpdfparaOff}
+}
+%    \end{macrocode}
+%
+%    Not sure why that was set in a test file, mistake?
+%    \begin{macrocode}
+%\AddToHook{package/amsmath/after}{
+%  \RegisterMathEnvironment{smallmatrix}
+%}  
+%    \end{macrocode}
+%    
+%    \begin{macrocode}
+\AddToHook{package/cases/after}{
+  \RegisterMathEnvironment{subnumcases}
+}  
```

Comment:
Good question. Before that, though, perhaps worth working out where is best to 'fix' the fact that `cases` can't be loaded after `amsmath`. I will move the load line to be with the one above, so the issue is collected in one place.

Reply:
I would say this is one of the topics we need a policy on once we see how much it involves either way. My expectation would be that for a set of "core" packages (where core needs some evaluation) we could do the registration using a scheme like the above initially, but long term I would expect to offer an interface and urge package owners to provide the registration themselves. So it is a bit like a first-aid for me: kick it off so that things work but get it midterm directly into the packages.